status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.181.31",
-    "commit-sha1": "6e056348e6d28f962167118612826f1ef0e47b22",
-    "src-sha256": "1qvfwk28sg93basjzy8r55qz8pk9xg0h7kxv5ykxkbfh4q17a1ns"
+    "version": "develop",
```

Comment:
If the capabilities you need are in status-go `develop`, then the PR should point to a tag. If there's no tag (quite common when desktop devs merge) you can push yourself or ask the developer that introduced the revision you need to do it.